lua/lazyvim/plugins/extras/lang/typst.lua

```diff
@@ -0,0 +1,36 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "typst" },
+    })
+  end,
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = { ensure_installed = { "typst" } },
+  },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        typst_lsp = {},
+      },
+    },
+  },
+  -- Typst preview
+  {
+    "chomosuke/typst-preview.nvim",
+    ft = "typst",
+    version = "0.3.*",
+    build = function()
+      require("typst-preview").update()
```

Comment:
I don't know if I should do this here
```suggestion
      require("lazy").load({ plugins = { "typst-preview.nvim" } }).update()
```

or
```suggestion
      require("lazy").load({ plugins = { "typst-preview" } }).update()
```

Please advise

Reply:
Why not just `require("typst-preview").update()` as you already have it? Did you encounter any problems?

Reply:
No, but the `markdown-preview` stuff uses the suggested approaches.
Just trying to know which is more "LazyVim idiomatic".

Reply:
`markdown-preview` uses this approach because the plugin doesn't have a Lua `setup` function and the plugin is lazy-loaded and the command `mkdp#util#install` is a vim command that is not available if the plugin is not loaded. I don't believe this is needed here and you should just leave it as you originally have, unless you're experiencing any problems. Requiring the module should also load the plugin and call the `update` function if I'm not mistaken.